volatility/framework/plugins/windows/hashdump.py

```diff
@@ -0,0 +1,346 @@
+
+from volatility.framework import interfaces, renderers
+from volatility.framework.configuration import requirements
+from volatility.framework.renderers import format_hints
+from volatility.plugins.windows.registry import hivelist
+from struct import unpack, pack
+from Crypto.Hash import MD5, MD4
+from Crypto.Cipher import ARC4, DES, AES
+import hashlib
+import binascii
+
+
+class Hashdump(interfaces.plugins.PluginInterface):
+    """Dumps user hashes from memory"""
+    @classmethod
+    def get_requirements(cls):
+        return [requirements.TranslationLayerRequirement(name = 'primary',
+                                                     description = 'Memory layer for the kernel',
+                                                     architectures = ["Intel32", "Intel64"]),
+            requirements.SymbolTableRequirement(name = "nt_symbols",
+                                                description = "Windows kernel symbols"),
+            requirements.PluginRequirement(name = 'hivelist', plugin = hivelist.HiveList, version = (1, 0, 0))
+            ]
+
+    odd_parity = [
```

Comment:
The entries in this table feel like they should be calculable?  Do you know how were they generated?  Not a show stopper but big chunks of hard coded data don't sit so well in the middle code...  5:S